pdf/config.json5

```diff
@@ -150,6 +150,10 @@
             parse: {
                 type: 'regex-counter',
                 stdstar: {
+                    '^\\s*(/opt|/lib|[#]FailureMessage).*$': {
+                        nameGroup: 'stack trace collapsed'
```

Comment:
I think some special characters should wrap this type of nameGroup.... e.g., `<stack trace collapsed>`. Otherwise it looks like pdfium output contained "stack trace collapsed".

Reply:
Or, maybe `FAW: stack trace collapsed" ?